src/mug_ffi.erl

```diff
@@ -1,6 +1,17 @@
 -module(mug_ffi).
 
--export([send/2, shutdown/1, coerce/1]).
+-export([connect_ip/4, connect_socket/3, send/2, shutdown/1, coerce/1]).
+
+connect_ip(Host, Port, Opts, Timeout) ->
+    gen_tcp:connect(Host, Port, Opts, Timeout).
+
+connect_socket(Host0, Opts, Timeout) ->
+    Host1 = case Host0 of
+        "/" ++ _Path -> {local, Host0};
+        "@" ++ Path -> {local, [0|Path]}
+    end,
```

Comment:
This will crash if the host doesn't start with `/` or `@`. Do we not want support relative paths to the socket?